src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -37,6 +37,7 @@ export interface Publication {
   contact: PublicationContact;
   methodology: {
     id: string;
+    title: string;
```

Comment:
We have a structure called IdTitlePair that you could probably reuse here

Reply:
unfortunately `IdTitlePair` is located in `@admin` and here we're making changes in the `@common` sub-project